lib/clients/incidents_client.dart

```diff
@@ -12,11 +12,19 @@ class IncidentsClient extends HTTPClient {
     return responseJSON.map((p) => new Incident.fromJson(p)).toList();
   }
 
+
   Future<List<Incident>> getIncidents() async {
     String pageID = await this.getRequestPageId();
-    List responseJSON = await this.requestGet('/pages/$pageID/incidents');
+    List responseJSON =
+        await this.requestGet('/pages/$pageID/incidents?limit=5');
```

Comment:
I don't think this should be changed

Reply:
Yeah sorry for that, I was trying for pagination then I forgot to remove.

Reply:
No worries! :)